app/controllers/attachments_controller.rb

```diff
@@ -9,18 +11,31 @@ def new
 	end
 
 	def create
-		@attachment = Attachment.create( attachment_params )
-
-		if @attachment.save!
+		begin
+			@attachment = Attachment.new(attachment_params)
+			@attachment.save!
 			flash[:success] = "File uploaded!"
```

Comment:
Workaround using pre-validations. It seems like Paperclip is bypassing standard error workflow.